wiki/Skinning/de.md

```diff
@@ -1,28 +1,28 @@
 # Skinning
 
-Skinning ist eine der Hauptfunktionen von osu! und ermöglicht Spielern den Standard-Skin ihren eigenen Wünschen entsprechend anzupassen oder ihre eigenen Skins zu erstellen! Deshalb gibt es Skins in allen Formen und Farben: sei es spaßeshalber, fokussiert auf "pro-Spielbarkeit", Minimalismus, oder nahezu unspielbar.
+Skinning ist eine der Hauptfunktionen von osu! und ermöglicht Spielern den Standard-Skin ihren eigenen Wünschen entsprechend anzupassen oder ihre eigenen Skins zu erstellen! Deshalb gibt es Skins in allen Formen und Farben: sei es spaßeshalber, mit Fokus auf professioneller Spielbarkeit, Minimalismus, oder nahezu unspielbar.
```

Comment:
Alternative:

```suggestion
Skinning ist eine der Hauptfunktionen von osu! und ermöglicht Spielern den Standard-Skin ihren eigenen Wünschen entsprechend anzupassen oder ihre eigenen Skins zu erstellen! Deshalb gibt es Skins in allen Formen und Farben: sei es spaßeshalber, mit Fokus auf E-Sports, Minimalismus, oder nahezu unspielbar.
```

Reply:
While we are at it, we could also remove the last comma in the paragraph.

Reply:
I prefer your changed version over "E-Sports"

Reply:
```suggestion
Skinning ist eine der Hauptfunktionen von osu! und ermöglicht Spielern den Standard-Skin ihren eigenen Wünschen entsprechend anzupassen oder ihre eigenen Skins zu erstellen! Deshalb gibt es Skins in allen Formen und Farben: sei es spaßeshalber, mit Fokus auf professioneller Spielbarkeit, Minimalismus oder nahezu unspielbar.
```